homework8/task01.py

```diff
@@ -0,0 +1,42 @@
+"""We have a file that works as key-value storage,
+each line is represented as key and value separated by = symbol, example:
+
+name=kek last_name=top song_name=shadilay power=9001
+
+Values can be strings or integer numbers.
+If a value can be treated both as a number and a string,
+it is treated as number.
+
+Write a wrapper class for this key value storage that works like this:
+
+storage = KeyValueStorage('path_to_file.txt')
+that has its keys and values accessible as collection items and as attributes.
+Example: storage['name'] # will be string 'kek'
+storage.song_name # will be 'shadilay'
+storage.power # will be integer 9001
+
+In case of attribute clash existing built-in attributes take precedence.
+In case when value cannot be assigned to an attribute
+(for example when there's a line 1=something) ValueError should be raised.
+File size is expected to be small,
+you are permitted to read it entirely into memory.
+"""
+
+
+class KeyValueStorage:
+    """wrapper class to access a storage-file by key-attributes"""
+    def __init__(self, path: str):
+        existing_builtins = set(self.__dict__.keys())
```

Comment:
```
print(existing_builtins)  # set()
```

if txt file contains: ```__class__=100```

u will get:  
```
kv = KeyValueStorage("test.txt")
kv.__class__, kv['__class__']   # KeyValueStorage, 100
```
it's confusing a bit.. u have 2 choises:
1. return the same values
2. dont return value for builtin atrrs in brackets